test/unit/gateways/worldpay_test.rb

```diff
@@ -330,6 +330,21 @@ def test_authorize_passes_stored_credential_options
     assert_success response
   end
 
+  def test_authorize_with_nt_passes_stored_credential_options
```

Comment:
Could you add another test were these GSF are present but also the hash `stored_credential`? I know that the fields in stored_credential would be send in the request but just for posterity

Reply:
Done!